integration-tests/olminstall/helpers/install_and_verify.py

```
@@ -0,0 +1,499 @@
+            sync_ready = int((r.stdout or "0").strip() or "0")
+        except ValueError:
+            sync_ready = 0
+        print(f"  nodes synced: {sync_ready}/{sync_desired}")
+        if sync_ready >= max(sync_desired, 1):
+            print(f"✓ quay.io/rhoai credentials synced to all {sync_desired} nodes")
+            return
+        time.sleep(10)
+
+    print(f"⚠ Syncer incomplete after 3m ({sync_ready}/{sync_desired} nodes) — proceeding")
+    subprocess.run(
+        ["oc", "get", "pods", "-n", "kube-system", "-l", "name=global-pull-secret-syncer", "--no-headers"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)